src/devtools/views/Elements/SelectedElement.js

```diff
@@ -74,6 +92,16 @@ export default function SelectedElement(_: Props) {
           </div>
         </div>
 
+        {canToggleSuspense && (
+          <Button
+            className={styles.IconButton}
+            onClick={toggleSuspense}
+            title="Toggle Suspense loading state"
+          >
+            <ButtonIcon type="toggle-suspense" />
+          </Button>
+        )}
```

Comment:
Wherever we end up placing this toggle, we should also:
* Show it as a toggle (with a clear on/off state).
* Only show it if the `Suspense` boundary has actually resolved. (Updates that trigger new suspends might be a little tricky, but maybe we can just ignore that.)